media-api/app/controllers/MediaApi.scala

```diff
@@ -299,16 +284,32 @@ object MediaApi extends Controller with ArgoHelpers {
     }
   }
 
-  private def getRelatedLabelsLink(mainLabel: String, selectedLabels: List[String] = Nil) = {
-    val uriTemplate = URITemplate(s"$rootUri/suggest/edits/labels/{label}/sibling-labels{?selectedLabels}")
-    val paramMap = Map(
-      "label" -> Some(mainLabel),
-      "selectedLabels" -> Some(selectedLabels.mkString(",")).filter(_.trim.nonEmpty)
-    )
-    val paramVars = paramMap.map { case (key, value) => key := value }.toSeq
-    val uri = uriTemplate expand (paramVars: _*)
+  private def getRelatedLabelsLink(searchParams: SearchParams) = {
+    // We search if we have a label in the search, take the first one and then look up it's
+    // siblings so that we can return them as "related labels"
+    val labels = searchParams.structuredQuery.flatMap {
+      // TODO: Use ImageFields for guard
+      case Match(field: SingleField, value:Words) if field.name == "userMetadata.labels" => Some(value.string)
+      case Match(field: SingleField, value:Phrase) if field.name == "userMetadata.labels" => Some(value.string)
```

Comment:
`case Match(field: SingleField, value: Value) if field.name == "userMetadata.labels" => Some(value.toString)`

Though you'd have to create another trait to for Words and Phrase (as DateRange also extends Value)... which is not worth it.